phoenix-core/src/it/java/org/apache/phoenix/end2end/LogicalTableNameExtendedIT.java

```diff
@@ -209,21 +209,26 @@ public void testHint() throws Exception {
             createIndexOnTable(conn, tableName, indexName2);
             populateTable(conn, tableName, 1, 2);
 
-            // Test hint
             String tableSelect = "SELECT V1,V2,V3 FROM " + tableName;
             ResultSet rs = conn.createStatement().executeQuery("EXPLAIN " + tableSelect);
-            assertEquals(true, QueryUtil.getExplainPlan(rs).contains(indexName));
+            String plan = QueryUtil.getExplainPlan(rs);
```

Comment:
why is this change required?

Reply:
There is assumption on the ordering of index PTables inside the parent table PTable. Optimizer can pick any of them but the test assumes only the first one in the list will get picked.